internal/lockservice/simpleLockService.go

```diff
@@ -1,15 +1,27 @@
 package lockservice
 
 import (
+	"fmt"
 	"sync"
+	"time"
 
 	"github.com/rs/zerolog"
 )
 
 // SafeLockMap is the lockserver's data structure
 type SafeLockMap struct {
-	LockMap map[string]string
-	Mutex   sync.Mutex
+	LockMap       map[string]*LockMapEntry
```

Comment:
`SafeLockMap.LockMap` seems weird.
Make the variable as `map` or something that doesnt repeat weirdly. (Kinda a Go standard)